### ISSUE TITLE
Publish 2.16 release

### ISSUE THREAD

**Issue by shane-axiom** [BobSimons/erddap#63]
There doesn't seem to be a [release published](https://github.com/BobSimons/erddap/releases) with the war and content zip for [v2.16](https://github.com/BobSimons/erddap/tree/v2.16). Can one be created? Thank you!

**Comment by BobSimons** [comment]
It has always been my standard procedure to install every new version at [https://coastwatch.pfeg.noaa.gov/erddap/index.html](https://coastwatch.pfeg.noaa.gov/erddap/index.html) to make sure it performs well for a week or so before making it available to others.  Since I started using Git(Hub), I also saved the release to Git(Hub) when installing it on that server. With the 2.16 release, that "week" is longer because I took some annual leave at the end of the year. 

I will probably finish the release (make the .war and content .zip available) today or tomorrow after I check a few things. There is also a possibility that I will release 2.17 instead (because of a new version of netcdf-java was released a few days after I released 2.16). If so, the release procedure would restart from the beginning.

Also, even after I make the .war accessible, I usually just notify a few people (notably, the people who requested certain changes). After another week, if all is well, I notify the ERDDAP Announcements Mailing list (anyone who wants to be added to that list can email me).

All of this is to try to catch problems before a new version is widely used. It is infrequent, but it happens. And there are often (there was this time) things that are hard to check without putting a new release on a public server.

I hope that explains the release process and the reason for the delay of the .war release.
